code/game/objects/items/weapons/tools/saws.dm

```diff
@@ -95,11 +95,11 @@
 
 
 /obj/item/weapon/tool/saw/plasma/turn_on()
-	.=..()
-	flick("plasma_saw_ignite", src)
-	playsound(get_turf(src), 'sound/weapons/saberon.ogg', 20, 1, -2)
-
+	.=.()
```

Comment:
this needs to be .=..()

i'm not sure why it compiles like this, but a dot has been deleted and it won't work right